bluez/tools/hciconfig.c

```diff
@@ -1844,12 +1879,19 @@ static void print_dev_hdr(struct hci_dev_info *di)
 
 	ba2str(&di->bdaddr, addr);
 
+	busname = ((di->type & 0x0f) <= HCI_MAX_BUS)
+		? hci_bustostr(di->type & 0x0f)
```

Comment:
It's a bit strange that `hci_bustostr()` returns `char*` rather than `const char*` but this has nothing to do with this PR